python/pylibcudf/pylibcudf/libcudf/unary.pxd

```diff
@@ -31,6 +31,7 @@ cdef extern from "cudf/unary.hpp" namespace "cudf" nogil:
         FLOOR
         ABS
         RINT
+        BIT_COUNT
```

Comment:
Can you add `BIT_COUNT` to our [type stub file](https://github.com/rapidsai/cudf/blob/branch-25.06/python/pylibcudf/pylibcudf/unary.pyi)?

Reply:
Done.